code/modules/mob/living/carbon/human/human.dm

```diff
@@ -292,114 +292,186 @@
-									return
-								usr << "<b>Name:</b> [R.fields["name"]]	<b>Criminal Status:</b> [R.fields["criminal"]]"
-								usr << "<b>Minor Crimes:</b>"
-								for(var/datum/data/crime/c in R.fields["mi_crim"])
-									usr << "<b>Crime:</b> [c.crimeName]"
-									usr << "<b>Details:</b> [c.crimeDetails]"
-									usr << "Added by [c.author] at [c.time]"
-									usr << "----------"
-								usr << "<b>Major Crimes:</b>"
-								for(var/datum/data/crime/c in R.fields["ma_crim"])
-									usr << "<b>Crime:</b> [c.crimeName]"
-									usr << "<b>Details:</b> [c.crimeDetails]"
-									usr << "Added by [c.author] at [c.time]"
-									usr << "----------"
-								usr << "<b>Notes:</b> [R.fields["notes"]]"
+							if(health != "Cancel")
+								R.fields["p_stat"] = health
+						if(href_list["m_stat"])
+							var/health = input(usr, "Specify a new mental status for this person.", "Medical HUD", R.fields["m_stat"]) in list("Stable", "*Watch*", "*Unstable*", "*Insane*", "Cancel")
+							if(usr.stat || H.weakened || H.stunned || H.restrained() || !istype(H.glasses, /obj/item/clothing/glasses/hud/health))
+								return
+							if(health != "Cancel")
+								R.fields["m_stat"] = health
+						if(href_list["evaluation"])
+							if(!getBruteLoss() && !getFireLoss() && !getOxyLoss() && getToxLoss() < 20)
+								usr << "<span class='notice'>No external injuries detected.</span><br>"
 								return
+							var/span = "notice"
+							var/status = ""
+							if(getBruteLoss())
+								usr << "<b>Physical trauma analysis:</b>"
+								for(var/obj/item/organ/limb/org in organs)
+									var/brutedamage = org.brute_dam
+									if(brutedamage > 0)
+										status = "received minor physical injuries."
+										span = "notice"
+									if(brutedamage > 20)
+										status = "been seriously damaged."
+										span = "danger"
+									if(brutedamage > 40)
+										status = "sustained major trauma!"
+										span = "userdanger"
+									if(brutedamage)
+										usr << "<span class='[span]'>The [org.getDisplayName()] appears to have [status]</span>"
+							if(getFireLoss())
+								usr << "<b>Analysis of skin burns:</b>"
+								for(var/obj/item/organ/limb/org in organs)
+									var/burndamage = org.burn_dam
+									if(burndamage > 0)
+										status = "signs of minor burns."
+										span = "notice"
+									if(burndamage > 20)
+										status = "serious burns."
+										span = "danger"
+									if(burndamage > 40)
+										status = "major burns!"
+										span = "userdanger"
+									if(burndamage)
+										usr << "<span class='[span]'>The [org.getDisplayName()] appears to have [status]</span>"
+							if(getOxyLoss())
+								usr << "<span class='danger'>Patient has signs of suffocation, emergency treatment may be required!</span>"
+							if(getToxLoss() > 20)
+								usr << "<span class='danger'>Gathered data is inconsistent with the analysis, possible cause: intoxication.</span>"

```

Comment:
This should probably be "toxin damage" or something instead of "intoxication", since that usually means "drunk"